README.md

```diff
@@ -249,7 +257,200 @@ AdyenCardComponent(
   },
 );
 ``` 
+<br>
+
+### Apple Pay Component with Sessions flow
+1. From your backend, make a [`sessions`](https://docs.adyen.com/api-explorer/Checkout/71/post/sessions) request. The response contains:
+- `sessionData`: the payment session data.
+- `id`: a unique identifier for the session data.
+- The request body.
```

Comment:
Do they also have to pass the request body to the app? That's how this sounds like.